### PR TITLE
Try to create MOJO_TMPDIR if it doesn't exist

### PR DESCRIPTION
**Pull request by nilxam**
Try to create MOJO_TMPDIR at least, but don't die if the tmpdir is
failed to create, just throw a error and continue, let MOJO to decide
where is the tmpdir.

### CODE REVIEW

**Comment by coolo** [comment]
looks like there is yet another broken cpan module ;(

File: lib/OpenQA/WebAPI.pm [hunk 1]
```diff
@@ -250,6 +251,13 @@ sub startup {
 
     unless ($ENV{MOJO_TMPDIR}) {
         $ENV{MOJO_TMPDIR} = ASSET_DIR . '/tmp';
+        # Try to create tmpdir if it doesn't exist but don't die if failed to create
+        if (!-e $ENV{MOJO_TMPDIR}) {
+            eval { make_path("$ENV{MOJO_TMPDIR}"); };
```
Comment (coolo) [inline lib/OpenQA/WebAPI.pm]: why would you quote that environment variable?

**Comment by nilxam** [comment]
it's wrong quote while doing test...

File: lib/OpenQA/WebAPI.pm (hunk 1 above)
Comment (nilxam) [inline lib/OpenQA/WebAPI.pm]: it's wrong quote while doing test...

**Pull request merged**